site/content/en/blog/releases/1.28.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "1.28.0 - OpenShift Support, Player Tracking Metrics and GKE Terraform Improvements"
+linkTitle: "1.28.0"
+date: "2022-12-06"
+---
+
+This is the 1.28.0 release of Agones.
+
+Highlights for this release include:
+* Support for OpenShift (where `OwnerReferencesPermissionEnforcement` is enabled)
+* Metrics for Player Tracking
+* GKE Terraform options for regional clusters and autoscaling
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.28.0" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
+
+- Add a FAQ entry describing when you would use Agones vs. StatefulSets [\#2770](https://github.com/googleforgames/agones/issues/2770)
+- Documentation: Kubernetes and Agones supported version matrix [\#2237](https://github.com/googleforgames/agones/issues/2237)
```

Comment:
Can we also please add in the enhancements (and any other items) we had from the RC release blog post too. Thanks!

Reply:
Done.